File: dotnet/test/Azure.Iot.Operations.Protocol.MetlTests/TestCasePushAcks.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

namespace Azure.Iot.Operations.Protocol.MetlTests
{
    public class TestCasePushAcks
    {
        public List<TestAckKind> Publish { get; set; } = new();

        public List<TestAckKind> Subscribe { get; set; } = new();

        public List<TestAckKind> Unsubscribe { get; set; } = new();
    }
}

File: go/test/protocol/test_case_push_acks.go
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
package protocol

type TestCasePushAcks struct {
	Publish     []TestAckKind `yaml:"publish"`
	Subscribe   []TestAckKind `yaml:"subscribe"`
	Unsubscribe []TestAckKind `yaml:"unsubscribe"`
}
